src/documentation_builder/lib/quattordocbuild/builder.py

```diff
@@ -0,0 +1,153 @@
+"""Build documentation from quattor sources."""
+
+import os
+import sys
+import jinja2
```

Comment:
I know it exists for python but it's own wesbite says it is not intended for production use yet. (version 0.1.post1 with no updates since 2015 on pypi). The server where the code should be is down (http://template-toolkit.org/svnweb/Template-Python) 
So I opted for jinja2 which is very very similar and is widely used and maintained.

Unless I'm missing something off course.